server/server_test.go

```diff
@@ -410,160 +407,6 @@ func TestCountOpenFiles(t *testing.T) {
 	}
 }
 
```

Comment:
I would really like to preserve this broadcast test using gossip instead of the http broadcaster
We could separate integration tests like this into server_integration_test.go and append the build tag
// +build integration

Then we would need to change Travis CI to run those using
go test -tags=integration